cli/tests/test_templater.rs

```diff
@@ -418,6 +418,30 @@ fn test_templater_signature() {
     insta::assert_snapshot!(render(r#"author"#), @"Test User <x@y>");
     insta::assert_snapshot!(render(r#"author.email()"#), @"x@y");
     insta::assert_snapshot!(render(r#"author.username()"#), @"x");
+
+    let (_, stderr) = test_env.jj_cmd_ok(&repo_path, &["--config-toml=user.name=''", "new"]);
+    insta::assert_snapshot!(stderr, @r###"
```

Comment:
I think this is covered in `test_global_opts.rs` (which I think was written before we had `--config-toml`, btw)

Reply:
I removed the `stderr` checks but kept the placeholder replacement checks.